pilot/pkg/features/pilot.go

```diff
@@ -120,6 +120,9 @@ var (
 	WorkloadEntryCrossCluster = env.Register("PILOT_ENABLE_CROSS_CLUSTER_WORKLOAD_ENTRY", true,
 		"If enabled, pilot will read WorkloadEntry from other clusters, selectable by Services in that cluster.").Get()
 
+	CentralIstiodAccess = env.Register("PILOT_CENTRAL_ISTIOD", true,
```

Comment:
I am not good at naming but this seems a bit confusing to me. In the past, we used 'external istiod' and 'central istiod' interchangably. We have an 'external istiod' flag already.

This isn't actually external istiod though, its just a single istiod shared between >1 clusters. I guess we call this 'primary remote' in our docs? https://istio.io/latest/docs/setup/install/multicluster/primary-remote/.

Maybe we just call it something like `PILOT_SUPPORT_REMOTE_CLIENTS` or something?